examples/error_handling/error_handling.py

```diff
@@ -0,0 +1,24 @@
+from dapr.clients import DaprClient
+from dapr.clients.exceptions import DaprGrpcError
+
+with DaprClient() as d:
+    storeName = 'statestore'
+
+    key = 'key||'
+    value = 'value_1'
+
+    # Wait for sidecar to be up within 5 seconds.
+    d.wait(5)
+
+    # Save single state.
+    try:
+        d.save_state(store_name=storeName, key=key, value=value)
+    except DaprGrpcError as err:
```

Comment:
Could you also add a nice `json()` method for `DaprGrpcError` which tries to give you a JSON version of all of the properties etc?

Reply:
Great idea! On it!